torch/_dynamo/variables/tensor.py

```diff
@@ -475,13 +180,55 @@ def call_method(
         from . import ConstantVariable, TupleVariable
 
         kwargs = dict(kwargs)
-
+        # print("CALLING TENSOR OP", name)
```

Comment:
?

Reply:
I find this extremely useful for debugging, and uncomment it regularly. I can remove it, if you like.

Reply:
Well, a comment like this isn't doing folks much good unless they find this spot, and if they found this spot it's easy enough to readd the print. So if we want to make it more generally useful, can we have this print by default in, e.g., logging.DEBUG mode?